source/_integrations/tuya.markdown

```diff
@@ -113,6 +113,13 @@ Click the created project to enter the `Project Overview` page and get the `Auth
 
 {% configuration_basic %}
 
+If no devices show up in Home Assistant:
+ - First make sure the devices show up in Tuya's cloud portal under the devices tab. 
+ - In the Tuya IoT configuration cloud portal you must NOT link your non-developer account under the "Users" tab.  Doing so will work, and you can even still add the devices under the devices tab, but the api will send 0 devices down to Home Assistant.  You must only link the account under the Devices->"Link Tuya App Account".  If it shows up on the users  tab be sure to delete it.
+ - Your region may not be correctly set
+ - Make sure your cloud plan does not need to be renewed (see error #28841002 on this page)
```

Comment:
What is `#28841002` referring too?

Reply:
If it expired and you are trying to add new I believe you will get "28841002: No permissions. Your subscription to cloud development plan has expired"   however if it is already in home assistant I do not believe an error is generated.